Default RootFS to host root

diff --git a/FEXCore/Source/Interface/Config/Config.json.in b/FEXCore/Source/Interface/Config/Config.json.in
index fa56db44c..ed4f79e46 100644
--- a/FEXCore/Source/Interface/Config/Config.json.in
+++ b/FEXCore/Source/Interface/Config/Config.json.in
@@ -137,7 +137,7 @@
     "Emulation": {
       "RootFS": {
         "Type": "str",
-        "Default": "",
+        "Default": "/",
         "Desc": [
           "Which Root filesystem prefix to use",
           "This can be a filesystem path",
